src/test/java/seedu/address/logic/parser/EditStudentCommandParserTest.java

```diff
@@ -137,32 +137,33 @@ public void parse_oneFieldSpecified_success() {
         // name
         Index targetIndex = INDEX_THIRD_PERSON;
         String userInput = targetIndex.getOneBased() + NAME_DESC_AMY;
-        EditPersonDescriptor descriptor = new EditPersonDescriptorBuilder().withName(VALID_NAME_AMY).build();
-        EditCommand expectedCommand = new EditCommand(targetIndex, descriptor);
+        EditStudentCommand.EditStudentDescriptor descriptor = new EditStudentDescriptorBuilder()
```

Comment:
should just use EditStudentDescriptor directly instead of referring to EditStudentCommand